pandas/core/indexes/datetimelike.py

```diff
@@ -121,6 +121,51 @@ def floor(self, freq):
 
     @Appender(_round_doc % "ceil")
     def ceil(self, freq):
+        """
+        Ceil the index to the specified freq.
+
+        Links to the available `frequency strings <http://pandas.
+        pydata.org/pandas-docs/stable/timeseries.html#offset-aliases>`_
+        and `frequency objects <http://pandas.pydata.org/pandas-docs/
+        stable/timeseries.html#dateoffset-objects>`_ .
+
+        Parameters
+        ----------
+        freq : freq string/object
```

Comment:
needs a description, type of parameters doesn't seem very standard.

Reply:
these can be string or pandas.DateOffset objects